tests/test_background_swap.py

```diff
@@ -0,0 +1,74 @@
+import os
+
+from torch.utils.data import DataLoader
+
+from continuum.transforms.custom import BackgroundSwap
+from continuum.datasets import CIFAR10, InMemoryDataset
+from continuum.datasets import MNIST
+import torchvision
+from continuum.scenarios import TransformationIncremental
+import pytest
+
+
+# Uncomment for debugging via image output
+# import matplotlib.pyplot as plt
+
+
+@pytest.mark.slow
+def test_background_swap_numpy():
+    """
+    Test background swap on a single ndarray input
+    """
+    mnist = MNIST("MNIST_DATA", download=True, train=True)
```

Comment:
Please use the custom path defined as env for all downloaded datasets, https://github.com/Continvvm/continuum/blob/master/tests/test_CIFAR100.py#L9